scripts/linters/pylint_extensions.py

```diff
@@ -543,7 +543,27 @@ class DocstringParameterChecker(checkers.BaseChecker):
             ' form: *args: list(*). Description.',
             'malformed-args-argument',
             'The parameter is incorrectly formatted.'
-        )
+        ),
+        'W9039': (
+            'Files must have a single newline below args in doc string.',
+            'single-space-below-args',
+            'Please enter a single newline below args in doc string.'
+        ),
+        'W9040': (
+            'Files must have a single newline below returns in doc string.',
+            'single-space-below-returns',
+            'Please enter a single newline below returns in doc string.'
+        ),
+        'W9041': (
+            'Files must have a single newline below raises in doc string.',
+            'single-space-below-raises',
+            'Please enter a single newline below raises in doc string.'
+        ),
+        'W9042': (
+            'Files must have a single newline below yield in doc string.',
+            'single-space-below-yield',
+            'Please enter a single newline below yield in doc string.'
+        ),
```

Comment:
This all looks duplicate, can we have just one message:

```
'Please add a single newline below %s in the docstring.', [This message should be an instruction, right?]
'single-space-below-each-dostring-section', [or something around that]
.
```

Reply:
Makes sense. The docstring checks for spaces above each section similarly uses one warning per section. Following the comments below, if we were to merge spaces above/below checks, should these warnings be refactored into a single warning as well?

Reply:
It would be great if you can do it! :) [Optional as per this PR context]